pootle/apps/pootle_app/management/commands/__init__.py

```diff
@@ -136,7 +136,7 @@ def handle_all(self, **options):
             project_query = project_query.filter(code__in=self.projects)
 
         for project in project_query.iterator():
-            tp_query = project.translationproject_set \
+            tp_query = project.translationproject_set.live() \
                               .order_by('language__code')
```

Comment:
without going through all the management commands - im not sure that none of them need to be able to see obsolete projects. Atm there is no tests - so we have no way of auto-knowing.

If you are confident that *none* of the management commands will be -vely affected - then gtm